swagger.yaml

```diff
@@ -88,27 +127,53 @@ paths:
         200:
           description: OK
           schema:
-            $ref: "#/definitions/DepartmentResponse"
+            $ref: "#/definitions/SearchResponse"
         400:
           description: Query term not specified
         500:
           description: Internal server error
 
-  /health:
+  /timeseries/{cdid}:
     get:
       tags:
-        - health
-      summary: "Health check API"
-      description: "API to query the app for current health status"
+        - timeseries
+      summary: "Query for timeseries data"
+      description: "API to query the app for timeseries by id"
+      parameters:
+        - in: query
+            name: cdid
```

Comment:
`cdid` should either be a query or a path parameter, but not both. 
If it's a query, then the path should be `/timeseries`. 
If it's part of the path (`/timeseries/{cdid}`) then lines 142 to 144 should be: 
...
      parameters:
        - in: path
            name: cdid
...

Here is [an example of when to use the `query` option is for pagination ](https://github.com/cadmiumcat/books-api/blob/3539b80a8a3ea127d2565b705a1c8d8d1baf0883/swagger.yml#L46)